tests/network/network_service/test_service_config_virtctl.py

```
@@ -0,0 +1,49 @@
+
+from tests.network.network_service.libservice import (
+    SERVICE_IP_FAMILY_POLICY_PREFER_DUAL_STACK,
+    SERVICE_IP_FAMILY_POLICY_REQUIRE_DUAL_STACK,
+    SERVICE_IP_FAMILY_POLICY_SINGLE_STACK,
+    assert_svc_ip_params,
+)
+
+
+class TestServiceConfigurationViaVirtctl:
+    @pytest.mark.parametrize(
+        "virtctl_expose_service, expected_num_families_in_service, ip_family_policy",
```

Ruff (0.14.13)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)